### PR TITLE
updating dependencies

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

**Comment by brettsam** [comment]
Can everyone double-check that this looks correct?

**Comment by pragnagopa** [comment]
- @maiqbal11 - Any updates for the python worker?
- @tylerl0706 / @daxian-dbw - Any updates for powershell worker?

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 1]
```diff
@@ -71,10 +71,10 @@
     <PackageReference Include="KubeClient" Version="2.1.5" />
     <PackageReference Include="KubeClient.Extensions.KubeConfig" Version="2.1.5" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.1.1" />
-    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.1.1-beta03-10111" />
-    <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.0.0-beta4-10082" />
+    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.2.0-beta01-SNAPSHOT" />
```
Comment (ahmelsayed) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj]: sorry, is it supposed to stay "SNAPSHOT" in the version?

**Comment by ahmelsayed** [comment]
@brettsam Can you also update [the minor version](https://github.com/Azure/azure-functions-core-tools/blob/83d80cbb818cb9d2023e6c7eed47ea048f107d89/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj#L9) of the CLI to 2

**Comment by brettsam** [comment]
Oh shoot -- I did it already. Let me know if I should back it out.

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj (hunk 1 above)
Comment (brettsam) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj]: I believe that's right -- @pragnagopa?

**Comment by ahmelsayed** [comment]
no, it's fine then, I just reset the build number on appveyor.

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj (hunk 1 above)
Comment (ahmelsayed) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj]: okay, just making sure :)

**Comment by maiqbal11** [comment]
The current Python worker version should be good for this release

**Comment by daxian-dbw** [comment]
@pragnagopa We have a few fixes in PowerShell worker, but no major changes. We can catch the next train.

**Pull request merged**